specification/chaos/Chaos.Management/client.tsp

```diff
@@ -41,6 +41,10 @@ using Microsoft.Chaos;
 
 // java
 @@clientName(Microsoft.Chaos, "ChaosManagementClient", "java");
+@@clientName(Azure.ResourceManager.Foundations.ManagedServiceIdentity,
+  "ResourceIdentity",
```

Comment:
I didn't rename `ManagedServiceIdentityType` -> `ResourceIdentityType`, as
1. there is another union already had this name in mgmt core
2. the class previously was `enum`, so it breaks anyway